kits/igor/igor2DemoIntegrated.m

```diff
@@ -135,6 +137,28 @@ function igor2DemoIntegrated( cam_module )
 % Wait to start
 while true
     fprintf('Paused. Click B3 button to start, B1 button to quit matlab...\n');  
+    % Set the state of the controller
+    cmdIO.a3 = 0;  % Set to snap
+    cmdIO.a6 = 0;  % Set to snap
+    cmdIO.e3 = 1;  % Highlight start button
+    cmdIO.e4 = 0;  % Don't highlight stop button
+    cmdIO.f5 = -1;  % Gains slider all the way down
+    controllerColor = 'b';
```

Comment:
:+1: This is awesome -- we should duplicate this (and below) controller config on the python API Igor example, @rLinks234